tests/test_meshtastic_utils_connect_paths.py

```
@@ -39,6 +40,91 @@ def test_connect_meshtastic_returns_existing_client(reset_meshtastic_globals):
     mock_tcp.assert_not_called()
 
 
+def test_connect_meshtastic_serializes_concurrent_connect_attempts(
+    reset_meshtastic_globals,
```

Ruff (0.15.9)

[warning] Unused function argument: `reset_meshtastic_globals`

(ARG001)